timemachine/fe/rbfe.py

```diff
@@ -160,12 +161,19 @@ def setup_initial_states(
     lambda_schedule: Union[NDArray, Sequence[float]],
     seed: int,
     min_cutoff: float = 0.7,
-) -> Tuple[List[InitialState], Callable[[float], InitialState]]:
+) -> List[InitialState]:
     """
-    Set up the initial states for a series of lambda values,
-    so we can (hopefully) bitwise recover the identical simulation to debug errors.
+    Given a sequence of lambda values, return a list of initial states.
+
+    Assumes lambda schedule is a monotonically increasing sequence in the closed interval [0, 1].
+
+    Notes
+    -----
+    * The InitialState objects can be used to recover a bitwise-identical simulation for debugging
 
-    Assumes lambda schedule is a monotonically increasing sequence in the closed interval [0,1].
+    * The returned function for constructing new optimized states uses pre-optimized states generated according to the
```

Comment:
remove, since this no longer returns a function

Reply:
Thanks, fixed in https://github.com/proteneer/timemachine/pull/959/commits/dc9c22f6f06fe3d3b8d5f32678372f592874adeb